src/Optimization/hiopHessianLowRank.cpp

```diff
@@ -196,6 +199,24 @@ hiopHessianLowRank::~hiopHessianLowRank()
   }
 }
 
+void hiopHessianLowRank::alloc_for_limited_mem(const size_type& mem_length)
+{
+  //note: St_ and Yt_ always have l_curr rows
+  if(l_curr == mem_length) {
```

Comment:
l_curr is a member variable.
All member variable names should end with trailing underscore to distingusish them from local variables. 
Maybe we should address this in another PR.